modules/client/test/integration/constants.ts

```diff
@@ -45,6 +51,10 @@ const grapqhlEndpoints = {
   failing: [{ url: "https://bad-url-gateway.io/" }],
 };
 
+export const ADDRESS_ONE = "0x0000000000000000000000000000000000000001";
+export const ADDRESS_TWO = "0x0000000000000000000000000000000000000002";
+export const ADDRESS_THREE = "0x0000000000000000000000000000000000000003";
+
 export const TEST_WALLET_ADDRESS = "0x8626f6940E2eb28930eFb4CeF49B2d1F2C9C1199";
 export const TEST_WALLET =
   "0xdf57089febbacf7ba0bc227dafbffa9fc08a93fdc68e1e42411a14efcf23656e";
```

Comment:
The `TEST_TOKEN_VOTING_DAO_ADDRESS`, `TEST_TOKEN_VOTING_PLUGIN_ADDRESS`... constants dont make sense anymore so why wee dont delete them?

Reply:
because even they don't fulfill a function anymore and can be any value that is a valid address. the tests still use them to issue the queries

Reply:
Well the full purpose of this constants was to point to daoAddresses, pluginAddresses... that existend on the version of subgraph used, since the whole subgraph is now mocked, i downt se why we still need it.

Maybe i'm missing something but if we change all appearences by ADDRESS_ONE we should be good anyways

Reply:
that is right. Let me change it